### ISSUE TITLE
Feature Request: Create subtasks in parenttask swimlane

### ISSUE THREAD

**Issue by BjoernLudwigPTB** [wekan/wekan#1953]
When creating subtasks from the according section inside an existing task, the subtask will be created in the upper most swimlane, where as usually the appropriate swimlane might be the same as the parent's task one, at least the way we structure our boards.

An option might be, to introduce another subtask related setting equivalent to the one for choosing a list for subtasks from other boards, to choose between:

- same as parent
- _swimlane1_
- _swimlane2_
- ...

where the actual swimlanes will be listed underneath the option 'same as parent'.

**Comment by Keelan** [comment]
Good feature. Running into this right now. Anyway to at least copy to the same level swimlane by default without create an whole UI option

**Comment by xet7** [comment]
@Keelan 

Please add screenshot mockup about what do you mean, and how it should work.

**Comment by TOSCom-DanielEder** [comment]
@xet7 I believe what Keelan is referring to, is making this default behaviour.
Right now a subtask will be created in the uppermost (or possible first/oldest) swimlane, ignoring where their parent is. It would probably make more sense for most use cases to have the subtask remain in the same lane as the parent task.

Example:
![image](https://user-images.githubusercontent.com/59686742/77628822-ddef5700-6f48-11ea-9b85-0a8f4e14146f.png)
Red arrow is the current behavior, green arrow is the desired behavior. 
A config option is naturally the best long term solution, but for the short-term this default behavior might be a lot easier to implement and go a long way in terms of usability.

**Comment by Keelan** [comment]
`const swimlaneId = card.swimlaneId;`
Wouldnt this set the subtask swimlane ID to that of the current card's swimlane ID (parent)?

Is `swimlaneId` persistent across boards? Or if you plan to drop subtasks on the same board as the parent. If so then this makes sense.

If not, wouldnt we have to do some some swimlane position lookup on the parent board and to drop the subtask into the nth swimlane on the sub board? Or try to match by Swimlane name

```javascript
      getDefaultSwimline() {
        let result = Swimlanes.findOne({ boardId: this._id });
        if (result === undefined) {
          Swimlanes.insert({
            title: TAPi18n.__('default'),
            boardId: this._id,
          });
          result = Swimlanes.findOne({ boardId: this._id });
        }
        return result;
      },
```

Why is some items called swimlane and some called swimline?

**Comment by TOSCom-DanielEder** [comment]
That is absolutely correct, I was basing this on a same-board scenario I had in mind, which obviously fails to handle two-board scenarios. 

I should say a matching by name is probably the most logical approach, as boards might have different amounts of swimlanes, and the swimlane position therefore might be off. If the name matching fails, falling back to the board's default lane should be a solid approach.

**Comment by BjoernLudwigPTB** [comment]
It's awesome to see so much activity around this proposal and @Keelan's idea of creating a more intuitive default behaviour as a first step. Thanks @TOSCom-DanielEder  and @Keelan !

**Comment by BjoernLudwigPTB** [comment]
We now have a more intuitive default behaviour resolved by #2976 described in the [according PR's comment](https://github.com/wekan/wekan/pull/2976#issuecomment-607693009). The UI setting described with a mock-up in the [opening comment](https://github.com/wekan/wekan/issues/1953#issue-370225557) is still desirable.
